### PR TITLE
New kurdish layouts

### PR DESCRIPTION
**Pull request by kurdikeyboard**
Added two new kurdish layouts Kurmanci and Standard kurdish layouts with some correction in codes of current layout.

### CODE REVIEW

**Comment by patrickgold** [comment]
Thanks for your PR! I will review, adjust (because of some syntax changes) and merge your layouts once my `subtype-specific-layouts` branch is merged in.

**Comment by patrickgold** [comment]
Ok so #561 is finally merged in and now it is clear where the layouts and config needs some adjustment to be successfully merged into master. Will add comments to the code lines to make it easier for you to fix the merge conflicts (not necessarily for git but for the FlorisBoard layout engine too).

File: app/src/main/assets/ime/config.json [hunk 1]
```diff
@@ -26,7 +26,9 @@
     "canadian_french": "Canadian French (QWERTY)",
     "greek": "Ελληνικά",
     "hebrew": "עברית",
-    "kurdish": "کوردی"
+    "kurdish": "کوردی",
+    "kurdish_standard": "کوردی (ستاندارد)",
+    "kurdish_kurmanci": "Kurdî"
```
Comment (patrickgold) [inline app/src/main/assets/ime/config.json:31]: Layout labels are now defined in the layout itself, thus the above two added lines should be removed.

File: app/src/main/assets/ime/config.json [hunk 2]
```diff
@@ -208,6 +210,11 @@
       "id": 2001,
       "languageTag": "ckb",
       "preferredLayout": "kurdish"
+    },
+    {
+    "id": 2301,
+    "languageTag": "ku",
+    "preferredLayout": "kurdish_kurmanci"
```
Comment (patrickgold) [inline app/src/main/assets/ime/config.json:217]: The preferred layout is now defined as seen in here: https://github.com/florisboard/florisboard/blob/7b25381850b5e5e3ca3c4c99f469b560ecddc423/app/src/main/assets/ime/config.json#L533-L538

File: app/src/main/assets/ime/text/characters/kurdish_kurmanci.json [hunk 3]
```diff
@@ -0,0 +1,45 @@
+{
+  "type": "characters",
+  "name": "kurdish_kurmanci",
```
Comment (patrickgold) [inline app/src/main/assets/ime/text/characters/kurdish_kurmanci.json:3]: Add the layout "label": "Your Custom Label" below the name, so the layout manager of FlorisBoard correctly displays the formatted layout name in the UI. Note that for existing layouts this change has been done automatically, you only have to edit all newly added layout files.

**Comment by patrickgold** [comment]
If the above issues are fixed and everything checks out then, I will happily merge your changes!

File: app/src/main/assets/ime/text/characters/kurdish_kurmanci.json (hunk 3 above)
Comment (kurdikeyboard) [reply app/src/main/assets/ime/text/characters/kurdish_kurmanci.json:3]: I added the label name thank you for your help :)

**Review by kurdikeyboard — COMMENTED** [review]
I fixed all things that you reqested to fix.

**Comment by patrickgold** [comment]
Thanks for fixing. There's still an issue with the config: You made a separate PR (#565) for adding the kurdish layouts to the config, it would be best for me if you'd include those changes in this PR, so I can properly test and merge the layouts.

**Comment by kurdikeyboard** [comment]
You welcome bro so what is next what you want todo? I think all required conflicts is fixed now you can merge itno the master code :)

**Comment by patrickgold** [comment]
I've manually resolved the merge conflict and updated the config, the Kurdi keyboard layout seems to be detected and working properly. Thanks again for your efforts! The new layouts will be released in the next beta v0.3.10-beta06!

**Comment by kurdikeyboard** [comment]
Thank you so much for support Kurdish language :)

**Pull request merged**